servicesContracts.py

```diff
@@ -1,28 +1,31 @@
 from typing import Any
-from servicesContracts.requestEthereum_service import RequestEthereumService
+from artifacts import *
+
 
-"""
-  getServiceFromAddress return the service of a coresponding currency contract address
-  :param  address:
-        The address of the currency contract
-"""
 def getServiceFromAddress(address: str) -> Any:
+    """
+    return the service of a corresponding currency contract address
+    :param  address:
+        The address of the currency contract
+    """
     if not address:
         return None
-    if isThisArtifact(Artifacts.requestEthereumArtifact, address):
+    if isThisArtifact(requestEthereumArtifact, address):
         return RequestEthereumService()
 
-"""
-  isThisArtifact return
-  :param  artifact:
-      RequestNetwork Artifact to use in its interactions with the Ethereum network
+
+def isThisArtifact(artifact, address: str) -> bool:
+    """
+    return True if any address in the network is sanitized address, otherwise it returns False
+    :param  artifact:
+        RequestNetwork Artifact to use in its interactions with the Ethereum network
     :param  address:
         The address of the currency contract
-"""
-def isThisArtifact(artifact, address: str) -> bool:
+    """
     if not address:
         return False
     sanitizedAdress = address.lower()
-    # assume artifact.networks is a dictionary
-    return any((value.address.lower() == sanitizedAdress if not value.address else False
-                for key,value in artifact.networks.items()))
\ No newline at end of file
+    for key, network in artifact["networks"].items():
```

Comment:
Since you don't use `key` you can just do `for network in artifact["networks"]:`

Reply:
You're right. I will change key to underscore "_". but I don't think ```for network in artifact["networks"]:``` would work because network is the value not the key.

Reply:
Oh whoops, my mistake. I would switch to `for network in artifact["networks"].values()`, just to get rid of `key` entirely.

Reply:
That works too. I will include it in the next commit.